pom.xml

```diff
@@ -194,18 +195,42 @@
                             <execution>
                                 <id>aggregate</id>
                                 <goals>
-                                    <goal>aggregate</goal>
-                                    <goal>aggregate-jar</goal>
+                                    <goal>aggregate-no-fork</goal>
+                                    <!--
+                                        until no-fork variant of jar is available, we'll package
+                                        directly with jar plugin
+                                    -->
+                                    <!--<goal>aggregate-jar</goal>-->
                                 </goals>
                                 <phase>pre-site</phase>
                                 <configuration>
                                     <!-- Specific configuration for the aggregate report -->
                                     <finalName>payara-5.192-SNAPSHOT</finalName>
+                                    <minmemory>512m</minmemory>
+                                    <maxmemory>2g</maxmemory>
```

Comment:
Does this override the settings from -Xmx & -Xms?

Reply:
I believe so. The javadoc process got more convenient heap